### PR TITLE
Episode 04

### PR DESCRIPTION
**Pull request by makraz**

### CODE REVIEW

File: blablas/ep4/index.md [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
+
+00:25 - Why do hackers always wear hoodies? Behind the stereotype?
+
+00:26 - What is your first project in entrepreneurship?
+
+00:31 - How to start a career in cybersecurity?
+
+00:33 - How you start your startup on Dubai?
+
+00:46 - Advice for someone how will start a startup?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:39]: Advice for people who want to found a startup. "Please omit the question mark"

File: blablas/ep4/index.md [hunk 2]
```diff
@@ -1,16 +1,57 @@
...
+
+00:46 - A person who wants to start on entrepreneurship it is required to have knowledge and Capital?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:41]: What kind of knowledge and amount of capital required for someone to take on entrepreneurship?

File: blablas/ep4/index.md [hunk 3]
```diff
@@ -1,16 +1,57 @@
...
+
+00:51 - How did you manage to get the funds?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:43]: please omit "the"

File: blablas/ep4/index.md [hunk 4]
```diff
@@ -1,16 +1,57 @@
...
+
+00:59 - What are your future vision for your startup and what is your advice for people on will start on the cybersecurity?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:45]: What are your future plans, and what kind of advice could you give to someone who wants to get into cybersecurity?

File: blablas/ep4/index.md [hunk 5]
```diff
@@ -1,16 +1,57 @@
...
+
+[Mohamed Zakariae El Khdime](https://www.facebook.com/infom2z)
+
+## Notes
+
+00:01 - Who is Mohamed Zakariae El Khdime?
+
+00:03 - What is your academic background?
+
+00:06 - did you study on any public high school?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:25]: in**

File: blablas/ep4/index.md [hunk 6]
```diff
@@ -1,16 +1,57 @@
...
+
+00:20 - How did you change your profile from developement to security?
```
Comment (mouradaouinat) [inline blablas/ep4/index.md:27]: how did you switch from** "no need for profile"

**Comment by yjose** [comment]
@all-contributors please add @makraz  for content

**Pull request merged**